lib/ace/ext/papeeria/highlighter.coffee

```diff
@@ -19,18 +20,19 @@ define((require, exports, module) ->
             if pos.left && pos.right 
                 range = new Range(pos.left.row, pos.left.column, pos.right.row, pos.right.column + 1)
                 session.$bracketHighlightLeft = session.addMarker(range, "ace_error-marker", "text")
-
             if pos.left && !pos.right
                 rangeLeft = new Range(pos.left.row, pos.left.column, Infinity, Infinity)
                 session.$bracketHighlightLeft = session.addMarker(rangeLeft, "ace_error-marker", "text")
-
             if pos.right && !pos.left
                 rangeRight = new Range(0, 0, pos.right.row, pos.right.column + 1)
                 session.$bracketHighlightRight = session.addMarker(rangeRight, "ace_error-marker", "text")
+        return
 
     findSurroundingBrackets = (editor) ->
         position = editor.getCursorPosition()
         session = editor.getSession()
+        if session.getLine(position.row).charAt(position.column - 1) of session.$brackets
```

Comment:
@Pochatkin I just've fixed it the way which looks more correct, please take a look.

For the record, there is no conflict with default highlighter, because there is no highlighting involved in `findOpeningBracket`. The issue was that TokenIterator which is used in `findOpeingBracket` is positioned in slightly unexpected wa. Its current token is the one whih immediately precedes the cursor position.